ui/src/components/rachat.css

```diff
@@ -1,9 +1,7 @@
 .chat-widget-dragable {
   position: absolute;
-  height: 100px;
-  width: 100px;
```

Comment:
This `height` was the cause of the positioning issue.

Reply:
Ok, I see thanks !